components/Form/Select/Select.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { string, bool, func, number, shape, arrayOf, oneOfType, objectOf } from 'prop-types';
+import { ErrorMessage } from 'formik';
+import Alert from 'components/Alert/Alert';
+import Label from 'components/Form/Label/Label';
+import ThemedReactSelect from './ThemedReactSelect';
+import styles from './Select.css';
+
+class Select extends React.Component {
+  static propTypes = {
+    disabled: bool,
+    field: shape({
+      name: string.isRequired,
+      value: oneOfType([
+        string,
+        arrayOf(shape({ label: string.isRequired, value: string.isRequired })),
+      ]),
+    }).isRequired,
+    form: shape({
```

Comment:
```javascript
<Field
  type="email"
  name="email"
  label="Email*"
  component={Input}
  disabled={isSubmitting}
  autoComplete="username email"
/>
```

It's easier to understand the "why" when you see an implementation.

Basically, `FIeld` has required props which are passed along as an object (`field`). Other props get passed thru, even when they're not related to the component.

tl;dr kind of necessary rather than by choice. we could put the component in a render prop to pass the specific props, but it's nice having a catch-all since the potential props are large. I'm just tryna avoid documenting all of Formik's API.